ui/tests/acceptance/allocation-detail-test.js

```diff
@@ -279,7 +279,7 @@ module('Acceptance | allocation detail', function (hooks) {
       assert.equal(renderedPort.name, serverPort.Label);
       assert.equal(renderedPort.to, serverPort.To);
       assert.equal(
-        renderedPort.address,
+        renderedPort.address.replace(/\s/g, ''),
```

Comment:
There's nothing I can do about the extra spaces being added to handlebars as a result of prettier, however. So I'm not sure what I can do without spending a few hours to understand why.

Do you have any ideas?

Reply:
You can ignore parts of the code: https://prettier.io/docs/en/ignore.html#handlebars

Reply:
Ignoring prettier seems yucky. I decided to use a template helper to avoid the problem. However, I think @philrenaud and I should prioritize taking a look into our prettier and linting set-up when I'm back from vacation.